apps/fluent-tester/src/TestComponents/Avatar/BasicAvatar.tsx

```diff
@@ -89,6 +89,7 @@ export const StandardUsage: FunctionComponent = () => {
           size={imageSize === undefinedText ? undefined : imageSize}
           shape={isSquare ? 'square' : 'circular'}
           accessibilityLabel="Icon"
+          image={{ source: showImage ? { uri: satyaPhotoUrl } : undefined }}
```

Comment:
just checking, since here image type is ImageProps (https://reactnative.dev/docs/image#imagesource)

why to add a outer object as source?

Reply:
My bad, I checked the props deeply in the https://github.com/facebook/react-native/blob/131844f40e15359fedd87ca2716a2880e0dd5ce4/Libraries/Image/ImageProps.js file, they do have a source object there.

Just wanted to double check my understanding here. I might be worng, do correct me,

When we say image is of type ImageProps -> then it can event be { uri: 'someUrl' } which is a valid IamgeProps,
but it will not render the avatar here, because we are checking for the source attribute. I think we can specify that in the documentation to use the source attribute.

Some RN Image uses: https://reactnative.dev/docs/images